src/solvers/dg_simplices/dg.jl

```diff
@@ -27,6 +30,8 @@ const DGWeakForm{Dims, ElemType} = DG{<:RefElemData{Dims, ElemType}, Mortar,
 # this is necessary for pretty printing
 Base.real(rd::RefElemData{Dims, Elem, ApproxType, Nfaces, RealT}) where {Dims, Elem, ApproxType, Nfaces, RealT} = RealT
 
+eachdim(mesh::AbstractMeshData{Dim}, dg::DG{<:RefElemData{Dim}}, cache) where {Dim} = Base.OneTo(Dim)
```

Comment:
Suggestion:
```suggestion
eachdim(mesh::AbstractMeshData{Dim}) where {Dim} = Base.OneTo(Dim)
```
Do you use `ndims(mesh::AbstractMeshData)`? Looks like it might be a good opportunity to define a generic `eachdim(whatever) = Base.OneTo(ndims(whatever))`...

Reply:
I don't think I do. I may have thought `ndims(mesh)` was used by Trixi's `Base.show` for some reason. 

I'll define `eachdim` in terms of `ndims` then.

Reply:
Yeah, we use it mostly for `show` - but it's available and fast, so we can use it here :wink: